components/dropdown/style/index.less

```diff
@@ -49,7 +49,7 @@
     background-clip: padding-box;
 
     &-item,
-    &-submenu-title {
+    &-submenu {
```

Comment:
I compared Menu's style and Dropdown's style, finding that this line fixed the problem, without side effects.